### DISCUSSION TITLE
Unsupported GPU: "vfio-pci" in log

### DISCUSSION THREAD

**Discussion by masterwishx** [photoprism/photoprism#2101] [Q&A]
using lasted version in Unraid docker
i changed from (intel-graphics tensorflow-amd64-avx2) to (gpu tensorflow)
and having "Unsupported GPU: "vfio-pci" not sure how it was befor in older versions.
![image](https://user-images.githubusercontent.com/28630321/156525611-11f4fe48-5192-4496-abb0-5e2e20152f03.png)
![image](https://user-images.githubusercontent.com/28630321/156525793-dfd799e2-8ae3-410e-b98f-fb4f98ef2734.png)

**Comment by lastzero** [comment]
Running inside a VM? What GPU do you have? Did you share devices as shown in our Docker Compose examples? Anything broken? Looks like a warning only. 

Googling it may be a good starting point:

https://access.redhat.com/discussions/794963

**Reply by masterwishx** [reply]
> Running inside a VM? What GPU do you have? Did you share devices as shown in our Docker Compose examples?

No im running docker in Unraid, having intel iGPU in Unraid (Intel GPU TOP plugin installed ) + GeForce GT 620 in vfio passed throuth.

used befor   **PHOTOPRISM_INIT = intel-graphics tensorflow-amd64-avx2** 
and now in last version changed to **PHOTOPRISM_INIT = gpu tensorflow**     like in  Docker Compose examples .

posted in photos above, is it right config ?

**Reply by lastzero** [reply]
It's an updated Linux base image with updated and more packages installed. The init targets have been shortened, the long targets work as well for now.

**Comment by masterwishx** [comment]
also having warning in docker console : WARNING: library configuration mismatch
![image](https://user-images.githubusercontent.com/28630321/156562221-3e6ccafc-1bf5-4f97-a114-105f15dc6c2d.png)

**Reply by lastzero** [reply]
We are on vacation right now. Could you or someone do a bit of research which packages are needed, maybe depends on the specific Intel hardware? See related ticket in GitHub Issues. It was tested and seemed to work.

**Reply by masterwishx** [reply]
OK, Thanks i will try to find somethink and will post here ...
using Intel® Core™ i5-4440 CPU btw.

if i understood right now its autodetect finding gpu and tensorflow.
Please confirm that we should use now **PHOTOPRISM_INIT = gpu tensorflow** instead of old **PHOTOPRISM_INIT = intel-graphics tensorflow-amd64-avx2**
